SprawdzanieNIPu/ConsoleApplication1/NIP.cs

```diff
@@ -1,72 +1,111 @@
-﻿using System;
+﻿//-----------------------------------------------------------------------
+// <copyright file="NIP.cs" company="CompanyName">
+// Company copyright tag.
+// </copyright>
+//----------------------------------------------------------------------- 
 
 namespace ConsoleApplication1
-{
-    class NIP
-    {
-        public string numer;
+ {
+    using System;
 
-        public NIP(string nr)
-        {
-            this.numer = nr.Trim();
-        }
+    /// <summary>
+    /// Klasa służy do sprawdzenia, czy wprowadzony ciąg znaków jest poprawnym numerem NIP.
+    /// </summary>
+ internal class NIP
+ {
+    /// <summary>
+    /// Wprowadzony numer po usunięciu białych znaków
+    /// </summary>
+   private string numer;
 
-        public Boolean dlugoscOK
-        {
-            get
-            {
-                return this.numer.Length == 10;
-            }
-        }
+    /// <summary>
+    /// Initializes a new instance of the NIP class.
+    /// </summary>
+    /// <param name="nr">Wprowadzony ciąg znaków</param>
+   public NIP(string nr)
```

Comment:
@naleczynskia Chcę mieć całą weryfikację wprowadzonego ciągu znaków w jednej klasie. Czy to złe podejście?